docs/ref/models/querysets.txt

```diff
@@ -2988,6 +2988,15 @@ unique field in the order that is specified without conflicts. For example::
     ``order_by()`` clause will be ignored if it contains annotations, inherited
     fields, or lookups spanning relations.
 
+.. warning::
+
+   When using MySQL, ``QuerySet.update()`` may execute two separate SQL queries
+   (a ``SELECT`` followed by an ``UPDATE``) instead of a single ``UPDATE`` when
+   filtering on related tables. This behavior occurs because MySQL does not support
+   self-select updates. As a result, this can introduce race conditions if
+   concurrent changes occur between the ``SELECT`` and ``UPDATE`` queries.
```

Comment:
We use four spaces for indentation. Also, I think we can edit this for brevity.

```suggestion
    MySQL does not support self-select updates. As a result, on MySQL
    ``QuerySet.update()`` may execute a ``SELECT`` followed by an ``UPDATE``
    instead of a single ``UPDATE`` when filtering on related tables, which can
    introduce a race condition if concurrent changes occur between the queries.